### PR TITLE
Render title menu button and wire up statistics link

### PR DESCRIPTION
**Pull request by emil-furniss**
Adding the preview link will be a separate PR.
Also, I don't know why it marked the entire file `d2l-rubric-editor-header` as changed, but I've commented on the two small additions I made there.

### CODE REVIEW

File: editor/d2l-rubric-editor-header.html [hunk 1]
```diff
@@ -1,113 +1,118 @@
...
+				justify-content: space-between;
+				flex-wrap: wrap;
+			}
+
+			#title {
+				@apply --d2l-heading-1;
+				margin-top: 0;
+				margin-bottom: 0;
+				display: flex;
+				align-items: center;
```
Comment (emil-furniss) [inline editor/d2l-rubric-editor-header.html]: Addition 1: Set `display: flex` and `align-items: center` to align title and new menu button

File: editor/d2l-rubric-editor-header.html [hunk 2]
```diff
@@ -1,113 +1,118 @@
...
+
+			d2l-alert {
+				margin: 0 var(--d2l-rubric-editor-gutter-width);
+			}
+		</style>
+		<div id="rubric-editor-header">
+			<div id="rubric-editor-header-content">
+				<div id="title">
+					<slot name="title"></slot>
+					<slot name="title-dropdown-menu"></slot>
```
Comment (emil-furniss) [inline editor/d2l-rubric-editor-header.html]: Addition 2: New slot for dropdown menu (which is specified in `d2l-rubric-editor`)

File: editor/d2l-rubric-editor.html [hunk 3]
```diff
@@ -91,6 +102,21 @@
...
 			<div slot="title">[[localize('editRubric')]]</div>
+			<d2l-dropdown
+				id="title-dropdown"
+				slot="title-dropdown-menu"
+				on-d2l-menu-item-change="_handleStatusChange">
+				<d2l-button-icon class="d2l-dropdown-opener" icon="d2l-tier1:chevron-down" text="Bookmark"></d2l-button-icon>
+				<d2l-dropdown-menu>
+					<d2l-menu id="title-menu">
+						<d2l-menu-item
+							text="Statistics"
```
Comment (emil-furniss) [inline editor/d2l-rubric-editor.html]: Langterm will be added in next PR as well

**Comment by mdulat** [comment]
> Adding the preview link will be a separate PR.
> Also, I don't know why it marked the entire file `d2l-rubric-editor-header` as changed, but I've commented on the two small additions I made there.

Looks like there were whitespace changes, you can hide whitespace changes in the Diff settings when reviewing.

File: editor/d2l-rubric-editor.html [hunk 4]
```diff
@@ -91,6 +102,21 @@
 		<d2l-rubric-editor-header
 			id="rubric-header">
 			<div slot="title">[[localize('editRubric')]]</div>
+			<d2l-dropdown
+				id="title-dropdown"
+				slot="title-dropdown-menu"
+				on-d2l-menu-item-change="_handleStatusChange">
```
Comment (mdulat) [inline editor/d2l-rubric-editor.html]: probably don't need this handler here.

File: editor/d2l-rubric-editor.html [hunk 5]
```diff
@@ -91,6 +102,21 @@
...
+				<d2l-button-icon class="d2l-dropdown-opener" icon="d2l-tier1:chevron-down" text="Bookmark"></d2l-button-icon>
```
[the remark this answers is not in the document]
Comment (emil-furniss) [reply editor/d2l-rubric-editor.html]: Nope, will delete. Sorry for the mess...

[the remark this answers is not in the document]
Comment (emil-furniss) [reply editor/d2l-rubric-editor.html]: And I'll check the alignment and fix if necessary.

**Pull request merged**